### DISCUSSION TITLE
无法使用除手机以外设备扫码登录

### DISCUSSION THREAD

**Discussion by ajjdxns** [DreamVoid/MiraiMC#498] [问答 / Q&A] [answered]
会出现code=238，message=请使用手Q扫描二维码或短信登录。

**Comment by DreamVoid** [comment]
要不然呢？

**Comment by ajjdxns** [comment]
我手头上只有一台平板

**Reply by DreamVoid** [reply]
那你就问腾讯为什么不让你扫码吧

**Reply by ajjdxns** [reply]
手机试了，也不行

**Reply by ajjdxns** [reply]
后面还多了个或短信登录

**Reply by DreamVoid** [reply, marked as answer]
我给你个关键词，“签名服务”

**Reply by ajjdxns** [reply]
可是我不会啊

**Reply by ajjdxns** [reply]
![image](https://github.com/DreamVoid/MiraiMC/assets/136149181/8ef17246-da82-4395-bbbd-f29b059c6c19)

**Reply by ajjdxns** [reply]
我试了试，更离谱

**Reply by ajjdxns** [reply]
![image](https://github.com/DreamVoid/MiraiMC/assets/136149181/b61c8fd9-4ee2-4704-a42b-58179345dd18)
